--- pspp-0.10.2/perl-module/Makefile.PL~	2016-04-03 03:57:38.000000000 +0200
+++ pspp-0.10.2/perl-module/Makefile.PL	2017-07-02 16:14:11.818358611 +0200
@@ -4,7 +4,7 @@
 # the contents of the Makefile that is written.
 
 
-do 'pspp-module-config' || do {
+do './pspp-module-config' || do {
     my $build = prompt ("Enter the location of the build directory of the configured pspp source:", "" );
     my $src = $top_srcdir;
 
